tools/realsense-viewer/CMakeLists.txt

```diff
@@ -53,6 +53,12 @@ if(BUILD_GRAPHICAL_EXAMPLES)
     ../../common/os.cpp
 )
 
+SET(DELAYED 
```

Comment:
I think it should be  under ifdef _WIN_ clause here and in the other instances below

Reply:
Having the list will not affect Linux build. It is being used only under WIN clause.